### PR TITLE
feat: pull missing data from github graphql api

### PR DESCRIPTION
**Pull request by zackpollard**

### CODE REVIEW

File: backend/src/interfaces/github.interface.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Repository } from '@octokit/webhooks-types';
+import { GithubRepo } from '../constants';
```
Comment (jrasm91) [inline backend/src/interfaces/github.interface.ts:2]: Can we use absolute imports? I don't remember if we have that setup yet.

File: backend/package.json [hunk 2]
```diff
@@ -37,6 +37,9 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.633.0",
     "@influxdata/influxdb-client": "^1.34.0",
+    "@octokit/auth-app": "^7.1.3",
+    "@octokit/graphql": "^8.1.1",
+    "@octokit/types": "^13.6.2",
```
Comment (jrasm91) [inline backend/package.json:42]: This one at least can probably be a dev dependency, right?

**Review by jrasm91 — COMMENTED** [review]
Looking good

File: backend/src/interfaces/github.interface.ts (hunk 1 above)
Comment (zackpollard) [reply backend/src/interfaces/github.interface.ts:1]: Apparently yes, that is setup :slightly_smiling_face:

**Review by danieldietzler — APPROVED** [review]

File: backend/src/interfaces/github.interface.ts (hunk 1 above)
Comment (zackpollard) [reply backend/src/interfaces/github.interface.ts:1]: I changed it and pushed the change so I have no idea :laughing:

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**